app/models/config.py

```diff
@@ -297,6 +300,10 @@ class Config:
         {'name': 'Zimbabwe', 'value': 'countryZW'}
     ]
 
+    THEMES = list(set([theme_name.split('.css')[0]
+                       for theme_name in os.listdir('app/static/css/themes/') 
+                       if "-search.css" not in theme_name]))
+
```

Comment:
I don't think the cast to `list` is necessary here, since the iterator for themes in the `index.html` template will work fine when provided with a `set`. 

```suggestion
    THEMES = set([theme_name.split('.css')[0]
                       for theme_name in os.listdir('app/static/css/themes/') 
                       if "-search.css" not in theme_name])

```